plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -187,6 +204,11 @@ export class ExpressInstrumentation extends InstrumentationBase<
           .filter(path => path !== '/' && path !== '/*')
           .join('');
 
+        const activeContext = context.active();
+        if (isTracingSuppressed(activeContext)) {
+          return original.apply(this, arguments);
```

Comment:
I don't think this early exit here is correct. suppress tracing means don't create a span but I guess the layer handling and setting it on rpcMetaData should be still done.

Reply:
You're right, fixed it